config/schema_config.yaml

```diff
@@ -191,6 +191,16 @@ super enhancer:
   is_a: non coding element
   inherit_properties: true
 
+non coding rna:
+  represented_as: node
+  input_label: non_coding_rna
+  is_a: non coding element
+  inherit_properties: true
+  properties:
+    rna_type: str
+    rfam: str[]
+    go_term: str[]
```

Comment:
You already have `go rna` relationship defined in the schema, what's the purpose of this property?